util/microservice-sdk/js/lib/configManager/fileManager/Reader.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @module Reader reads the config files.
+ */
+const { readFileSync, existsSync } = require('fs');
+
+const Sanitizer = require('./Sanitizer');
+const Utils = require('../Utils');
+const { Logger } = require('../../logging/Logger');
+
+const logger = new Logger('microservice-sdk:config-manager-reader');
+
+// Default configuration file location
+const DefaultConfigLocation = './config/default.conf';
+
+/**
+ * Reads the default configuration file and returns an array of unparsed parameters.
+ *
+ * @returns {string[]}
+ * @throws
+ */
+const readDefaultConfig = () => {
+  if (existsSync(DefaultConfigLocation)) {
+    const data = readFileSync(DefaultConfigLocation).toString();
```

Comment:
Given that you are using lib 'fs', `DefaultConfigLocation` is relative to the current working directory (which is the directory that was current when you started node). If the node command is executed inside the project directory, I believe there is no problem in finding the file in the correct location, but otherwise, I think it is interesting to use an absolute path instead of a relative one.

In the example below, it is possible to obtain the absolute path of the initial script passed to the node command, even if the current shell directory is not inside the project folder:

```js
const path = require('path');
const mainDir = path.dirname(require.main.filename);
const DefaultConfigLocation = path.join(mainDir, 'config/default.conf');
```
https://nodejs.org/api/modules.html#modules_require_main